src/gifcodec.c

```diff
@@ -306,9 +302,9 @@ gdip_load_gif_image (void *stream, GpImage **image, BOOL from_file)
 	loop_counter = FALSE;
 
 	if (from_file) {
-		gif = DGifOpen(stream, &gdip_gif_fileinputfunc);
+		gif = DGifOpen(stream, &gdip_gif_fileinputfunc, NULL);
 	} else {
-		gif = DGifOpen (stream, &gdip_gif_inputfunc);
+		gif = DGifOpen (stream, &gdip_gif_inputfunc, NULL);
```

Comment:
these need the GIFLIB_MAJOR guards as well, or it will fail to compile on earlier giflibs.